lib/webpacker/manifest.rb

```diff
@@ -42,19 +42,36 @@ def find(name)
 
     def handle_missing_entry(name)
       raise Webpacker::Manifest::MissingEntryError,
-        "Can't find #{name} in #{config.public_manifest_path}. Is webpack still compiling?"
+        "Can't find #{name} in #{config.public_manifest_path}. Manifest contains:\n#{JSON.pretty_generate(@data)}"
+    end
+
+    def missing_file_from_manifest_error(bundle_name)
+      msg = <<-MSG
+Webpacker can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
+1. You are hot reloading.
+2. You want to set Configuration.compile to true for your environment.
+3. Webpack has not re-run to reflect updates.
+4. You have misconfigured Webpacker's config/webpacker.yml file.
+5. Your Webpack configuration is not creating a manifest.
+Your manifest contains:
+#{@data.to_json}
+    MSG
```

Comment:
@gauravtiwari @dhh Do we want a more detailed error message? or at least what I did on line 45 to output the manifest?